pdns/dnsparser.hh

```diff
@@ -220,7 +220,7 @@ public:
     return typeid(*this)==typeid(rhs) && this->getZoneRepresentation() == rhs.getZoneRepresentation();
   }
   
-  static shared_ptr<DNSRecordContent> unserialize(const DNSName& qname, uint16_t qtype, const string& serialized);
+  static shared_ptr<DNSRecordContent> deserialize(const DNSName& qname, uint16_t qtype, const string& serialized);
```

Comment:
I'm a bit worried about this change, as it might break a backend or feature that we are not currently building in CI.It's probably fine, though.

Reply:
Fine with me :)